common/unsafe/src/main/java/org/apache/spark/unsafe/types/CalendarInterval.java

```diff
@@ -160,7 +160,25 @@ public static CalendarInterval fromYearMonthString(String s) throws IllegalArgum
    * adapted from HiveIntervalDayTime.valueOf
    */
   public static CalendarInterval fromDayTimeString(String s) throws IllegalArgumentException {
-    CalendarInterval result = null;
+    return fromDayTimeString(s, "day", "second");
+  }
+
+  /**
+   * Parse dayTime string in form: [-]d HH:mm:ss.nnnnnnnnn and [-]HH:mm:ss.nnnnnnnnn
+   *
+   * adapted from HiveIntervalDayTime.valueOf.
+   * Below interval conversion patterns are supported:
+   * - DAY TO HOUR
+   * - DAY TO MINUTE
+   * - DAY TO SECOND
```

Comment:
This is not in the title.

Reply:
'DAY TO SECOND' is supported by this method. Should I remove this line or add this in title?

Reply:
Since this is supported, you only need to update the PR title.
I'm not sure the title space is enough, but please try~